core/src/main/java/com/sap/ai/sdk/core/commons/IterableStreamConverter.java

```diff
@@ -1,4 +1,4 @@
-package com.sap.ai.sdk.foundationmodels.openai;
```

Comment:
(Question)

`commons` (plural) looks surprising to me.
Are other libraries inside / outside of Cloud SDK doing the same? Was there some discussion? (Is there need to discuss that in Slack?)